[SOAR 1784, 1785] Improvements to connection and create address object action (#564)
rapid7/insightconnect-plugins: fortinet_fortigate/help.md

# Version History

> * 4.0.2 - Improve connection to allow for a Hostname with or without the https:// prefix | Improve Create Address Object action to allow for IPs and CIDRs as input

Reviewer: ```suggestion
* 4.0.2 - Support host URL in connection | Improve Create Address Object action to allow for IPs and CIDRs as input
```
